### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by ImDP**
This should work!

### CODE REVIEW

**Comment by tsidid** [comment]
Thanks very much Duncan, I appreciate your help.

**Comment by ImDP** [comment]
Make sure you "commit" the changes by saving and everything should work.

**Comment by ImDP** [comment]
I see you're in Ghana! I've heard great things about Ghana and have a friend here in Virginia from Accra! Great city!

Feel free to follow me and @ me for help anytime.

Cheers,
Duncan
________________________________

File: index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <html>
     <h1>Tsidi's Portfolio</h1>
-    <img src="./biopic.JPG" />
-</html>
\ No newline at end of file
+    <img src="tttt.jpg" />
```
Comment (tsidid) [inline index.html:3]: This looks like the solution.

Comment (ImDP) [reply index.html:3]: Yes, correct! Now you need to commit the change. it's still showing as biopic in index.html

**Pull request closed without merging**